compiler-rt/test/asan/TestCases/suppressions-alloc-dealloc-mismatch.cpp

```diff
@@ -0,0 +1,33 @@
+// Check that without suppressions, we catch the issue.
+// RUN: %clangxx_asan -O0 %s -o %t
+// RUN: not %run %t 2>&1 | FileCheck --check-prefix=CHECK-CRASH %s
+
+// RUN: echo "alloc_dealloc_mismatch:function" > %t.supp
+// RUN: %clangxx_asan -O0 %s -o %t && %env_asan_opts=suppressions='"%t.supp"' %run %t 2>&1 | FileCheck --check-prefix=CHECK-IGNORE %s
+// RUN: %clangxx_asan -O3 %s -o %t && %env_asan_opts=suppressions='"%t.supp"' %run %t 2>&1 | FileCheck --check-prefix=CHECK-IGNORE %s
+
+// XFAIL: android
```

Comment:
Any comment worth on why it's XFAILd for android?

Reply:
Ah, this is copied from another test case, which I used to start this one.  Will clean up.

Reply:
Probably Android test harness does not copy suppression onto device.
I assume it will fail here too, comment here will be helpful.

Reply:
Should I just remove this?  Or keep w/ a comment about assuming a failure would happen?  I'm not sure how to test on android.

Reply:
Ahh, I did remove it in the most recent update -- not sure if this was the right thing to do.